core/src/main/java/nl/nn/adapterframework/core/PipeLineExit.java

```diff
@@ -53,21 +60,26 @@ public boolean isSuccessExit() {
 	}
 
 	/**
-	 * Name of the pipeline exit
+	 * The name of the Exit that can be referenced by the Forward of a Pipe. When a Pipeline doesn't have an Exits
+	 * element configured it will be initialized with one Exit having name READY (and state SUCCESS)
 	 * @ff.mandatory
 	 */
-	public void setPath(String newPath) {
-		path = newPath;
+	public void setName(String name) {
+		this.name = name;
 	}
-	@Override
-	// getName() is required by {@link IForwardTarget}. It is required that it returns the path,
-	// this way PipeForwards can be resolved to either Pipes or PipeLineExits.
-	public String getName() {
-		return getPath();
+
+	/**
+	 * The name of the Exit that can be referenced by the Forward of a Pipe. When a Pipeline doesn't have an Exits
+	 * element configured it will be initialized with one Exit having name READY (and state SUCCESS)
+	 * @deprecated
+	 */
+	public void setPath(String path) {
+		name = path;
```

Comment:
Hier graag setName() aanroepen, in plaats van waarde te assignen aan veld.

Reply:
Done